src/components/AppSelector.tsx

```diff
@@ -447,7 +478,12 @@ export function AppSelector({
                             categoryEmojis[option.category as AppCategory]
                           )}
                         </span>
-                        <span className="truncate">{getOptionDetails(option).text}</span>
+                        <span className="truncate">
+                          {isCategory
```

Comment:
I like to keep the jsx free of logic and just layout. I'd recommend putting this ternary into a variable that's used here instead